src/components/common/ImportModal.tsx

```diff
@@ -137,16 +149,9 @@ export default function ImportSpeciesModal(props: ImportSpeciesModalProps): JSX.
     if (setCheckDataModalOpen) {
       setCheckDataModalOpen(true);
     }
-  };
-
-  const downloadCsvTemplate = async () => {
-    const apiResponse = await templateApi();
-    const csvContent = 'data:text/csv;charset=utf-8,' + apiResponse;
-    const encodedUri = encodeURI(csvContent);
-    const link = document.createElement('a');
-    link.setAttribute('href', encodedUri);
-    link.setAttribute('download', `template.csv`);
-    link.click();
+    if (reloadData) {
```

Comment:
Was this refreshing before this change?

Reply:
here we want to refresh the entire state including hasSeedBanks, hasSpecies etc which is passed in from App.tsx
.. wasn't a requirement before this feature